dozer-types/src/json_types.rs

```diff
@@ -114,7 +114,13 @@ pub fn field_to_json_value(field: Field) -> JsonValue {
         Field::String(s) => s.into(),
         Field::Text(n) => n.into(),
         Field::Binary(b) => b.into(),
-        Field::Decimal(n) => n.to_string().into(),
+        Field::Decimal(n) => serde_json_to_json_value(
```

Comment:
Instead of turning it into a string, this will make it lose precision. I think it's better to just turn it into a string.

Reply:
Not sure I understand your first sentence. Did you mean turning into a number?